src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+.main {
+  margin: 0;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  gap: 46px 48px;
```

Comment:
The `grid-template-columns` property with a single value of `200px` is not necessary if you are just creating a single column. You can remove this line as it will be overridden by the media queries anyway.